python/ingestion/standardized_columns.py

```diff
@@ -127,6 +127,15 @@
 HIV_DIAGNOSES_PER_100K = 'hiv_diagnoses_per_100k'
 HIV_DIAGNOSES_PCT_INEQUITY = "hiv_diagnoses_pct_relative_inequity"
 HIV_DIAGNOSES_PCT_SHARE = 'hiv_diagnoses_pct_share'
+HIV_DEATHS = 'hiv_deaths'
```

Comment:
For similar column names I have tried to use a `prefix` and `suffix` standard, and then to tie them together with the `generate_column_name` function. You can see examples in the `uhc.py` and `cdc_restricted.py` files